dcc-storage-client/pom.xml

```diff
@@ -160,9 +160,12 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
         <artifactId>maven-jar-plugin</artifactId>
         <configuration>
           <archive>
-            <manifestEntries>
-              <Implementation-Version>${project.version}</Implementation-Version>
-            </manifestEntries>
+
+            <manifest>
+              <mainClass>org.icgc.dcc.storage.client.ClientMain</mainClass>

```

Comment:
This is not the main class and will break things. I suggest reverting the changes to this file.